editor/components/block-list/style.scss

```diff
@@ -396,6 +430,21 @@
 	}
 }
 
+.editor-block-list__layout > .editor-block-list__insertion-point {
+	position: relative;
+	margin-top: -15px;
```

Comment:
This causes misalignment of the first block, overlapping the title and putting the text caret at the wrong position when clicking within the writing prompt:

![prompt](https://user-images.githubusercontent.com/1779930/36690748-424a491a-1b01-11e8-878d-73a05bc089bd.gif)

The line also extends beyond the correct bounds of the block list / title:

![image](https://user-images.githubusercontent.com/1779930/36690694-2196e3ea-1b01-11e8-99b8-5dc7a29d4d86.png)

Reply:
Also noting that the misalignment impacts nested blocks, and should be accounted for:

![image](https://user-images.githubusercontent.com/1779930/36692883-69229ec4-1b07-11e8-9880-252639356060.png)

(The tops of the squares should align)